pizzabot.js

```diff
@@ -0,0 +1,83 @@
+const vegetarian = "Vegetarian Pizza";
+const hawaiian = "Hawaiian Pizza";
+const pepperoni = "Pepperoni Pizza";
+const pizzaPrice = 80;
+
+document.getElementById(
+  "botBubble"
+).innerHTML = `Hey! Happy to get your order started. On our menu we have ${vegetarian}, ${hawaiian}, and ${pepperoni}.`;
+
+document.getElementById("userBubble").innerHTML =
+  "Ok! (Click to start your order.)";
+
+document.getElementById("userBubble").style.cursor = "pointer";
```

Comment:
CSS in JS ⛔️
Same is addressed to lines below

Reply:
moving this styling to CSS.